api/ordinals.ts

```diff
@@ -4,14 +4,24 @@ import XordApiProvider from '../api/ordinals/provider';
 import { INSCRIPTION_REQUESTS_SERVICE_URL, ORDINALS_URL, XVERSE_API_BASE_URL, XVERSE_INSCRIBE_URL } from '../constant';
 import {
   Account,
+  AddressBundleResponse,
   Brc20HistoryTransactionData,
   BtcOrdinal,
+  Bundle,
+  BundleSatRange,
   FungibleToken,
   HiroApiBrc20TxHistoryResponse,
   Inscription,
   InscriptionRequestResponse,
   NetworkType,
+  RareSatsType,
+  RoadArmorRareSats,
+  RoadArmorRareSatsType,
```

Comment:
lol
The rare sat category is called Rodarmor 😋

Reply:
that is disappointing. Road Armor was cool